assets/components/LogViewer/CopyLogMessage.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    class="absolute -right-1 flex min-w-[0.98rem] items-start justify-end align-bottom hover:cursor-pointer"
+    v-if="message.trim() != ''"
+    title="Copy Log"
+  >
+    <span
+      class="rounded bg-slate-800/60 px-1.5 py-1 text-primary opacity-0 transition-opacity delay-500 duration-1000 hover:bg-slate-700 group-hover/entry:opacity-100"
+      @click="copyLogMessageToClipBoard()"
+    >
+      <carbon:copy-file />
+    </span>
+  </div>
+</template>
+
+<script lang="ts" setup>
+const { message } = defineProps<{
+  message: string;
+}>();
+
+const { showToast } = useToast();
+
+function copyLogMessageToClipBoard() {
+  navigator.clipboard.writeText(message);
+
+  showToast(
+    {
+      title: "Copied",
+      message: "Log message copied to clipboard",
```

Comment:
don't forget to move these to locale

Reply:
@amir20 I find that there is no option to use `i8n` inside `<script>`, am I not seeing properly?

Reply:
Should be able to do `const { t } = useI18n();`